examples/testapp/src/context/EIP1193ProviderContextProvider.tsx

```diff
@@ -0,0 +1,102 @@
+import { createCoinbaseWalletSDK as createCoinbaseWalletSDKHEAD } from '@coinbase/wallet-sdk';
+import { createCoinbaseWalletSDK as createCoinbaseWalletSDKLatest } from '@coinbase/wallet-sdk-latest';
+
+import React, { useEffect, useMemo } from 'react';
+import { DisconnectedAlert } from '../components/alerts/DisconnectedAlert';
+import { useEventListeners } from '../hooks/useEventListeners';
+import { useSpyOnDisconnectedError } from '../hooks/useSpyOnDisconnectedError';
+import { scwUrls } from '../store/config';
+import { cleanupSDKLocalStorage } from '../utils/cleanupSDKLocalStorage';
+import { useConfigParams } from './ConfigParamsContextProvider';
+
+type EIP1193ProviderContextProviderProps = {
+  children: React.ReactNode;
+};
+
+type EIP1193ProviderContextType = {
+  sdk:
+    | ReturnType<typeof createCoinbaseWalletSDKHEAD>
+    | ReturnType<typeof createCoinbaseWalletSDKLatest>;
+  provider: ReturnType<EIP1193ProviderContextType['sdk']['getProvider']>;
+};
+
+const EIP1193ProviderContext = React.createContext<EIP1193ProviderContextType | null>(null);
+
+export function EIP1193ProviderContextProvider({ children }: EIP1193ProviderContextProviderProps) {
+  const { option, version, scwUrl, config } = useConfigParams();
+  const { addEventListeners, removeEventListeners } = useEventListeners();
+  const {
+    spyOnDisconnectedError,
+    isOpen: isDisconnectedAlertOpen,
+    onClose: onDisconnectedAlertClose,
+  } = useSpyOnDisconnectedError();
+
+  const [sdk, setSdk] = React.useState(null);
+  const [provider, setProvider] = React.useState(null);
+
+  useEffect(() => {
+    cleanupSDKLocalStorage();
```

Comment:
Are we just wiping all storage on app load now?

Reply:
ahh good call out, switched to only clean up on config update at configprovider